src/index.js

```diff
@@ -1,11 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import App from './components/App';
+import rootReducer from './reducers';
+
+const defaultStore = {
+  books: [
+    {
+      id: parseInt(Math.random() * 1000, 10),
+      title: 'The Peaceful Warrior',
+      category: 'Biography',
+    },
+    {
+      id: parseInt(Math.random() * 1000, 10),
+      title: 'Harry Potter',
+      category: 'Kids',
+    },
+    {
+      id: parseInt(Math.random() * 1000, 10),
```

Comment:
`parseInt(Math.random() * 1000, 10)` it's being repeated here three times.

Wouldn't it be nice to extract it into a function and give an expressive name, for example, `generateId`.

That way the code is going to be more readable and we don't repeat ourselves, and if at some point we want to change the way the id is generated, we just have to modify it in one place.